unit-tests/unit-tests-common.h

```diff
@@ -858,6 +858,81 @@ inline rs2::depth_sensor restart_first_device_and_return_depth_sensor(const rs2:
     return depth_sensor;
 }
 
+inline rs2::device return_first_device_of_requeired_product( const rs2::context & ctx, int product )
+{
+    rs2::device_list devices_list = ctx.query_devices( product );
+    size_t device_count = devices_list.size();
+    REQUIRE( device_count > 0 );
```

Comment:
The unit-tests need to succeed if there are no devices, and exit -- they need to run in Travis.
Sync with Nir and Matan, see the e-mail from this morning: the Python and C++ names and behaviors should match.

Reply:
Also, can just use `! devices_list.empty()`

Reply:
Changed to exit( 0 ); 
The use of .empty() require explicit conversion to  std::vector<device>()